tests/test_constraints.py

```diff
@@ -178,7 +178,12 @@ def test_batched_constraints(backend):
     suggested_pars = [1.0] * 3 + [0.0] * 5  # 2 pois 5 norm
     constraint = poisson_constraint_combined(config)
     result = default_backend.astensor(
-        pyhf.tensorlib.tolist(constraint.logpdf(config.auxdata, suggested_pars))
+        pyhf.tensorlib.tolist(
+            constraint.logpdf(
+                pyhf.tensorlib.astensor(config.auxdata),
+                pyhf.tensorlib.astensor(suggested_pars),
```

Comment:
This and the following would then get changed as

```suggestion
                suggested_pars,
```

Though maybe your wrapping like this is to have visual parity with `pyhf.tensorlib.astensor(config.auxdata)` and follow being super explicit about what is happening in tests?

Reply:
^^